main.py

```diff
@@ -323,6 +323,74 @@ async def clear_scores() -> None:
 
     print(f"Deleted all clearable scores in {time.time() - start_time:.2f} seconds")
 
+CHAMPION_BADGE_ID = 67
+FORMER_CHAMPION_BADGE_ID = 66
+
+async def update_champion_badges() -> None:
+    for rx in (0, 1, 2):
+        if rx == 0:
+            redis_board = "leaderboard"
+            modes = ("std", "taiko", "ctb", "mania")
+        elif rx == 1:
+            redis_board = "relaxboard"
+            modes = ("std", "taiko", "ctb")
+        else:  # rx == 2:
+            redis_board = "autoboard"
+            modes = ("std",)
+
+        for mode in modes:
+            mode_int = mode + (4 * rx)
+            rank_key = f"ripple:{redis_board}:{mode}"
+
+            # this is updated in a cron so we need to also check current rank one
+            users_who_peaked_at_rank_one = list(
+                await db.fetchall(
+                    "SELECT DISTINCT user_id FROM user_profile_history WHERE mode = %s AND `rank` = 1",
```

Comment:
Could you check how many users are going to receive a badge that don't currently have it, from the switch from `manual -> auto based on user_profile_history data`?

Reply:
I think there's a bit to consider here re. cases where someone cheated their way to #1 then got unbanned -- are they really deserving of "former champion" in that case?

Reply:
> I think there's a bit to consider here re. cases where someone cheated their way to https://github.com/osuAkatsuki/new-cron/pull/1 then got banned/unbanned

if we're not wiping this table when wiping a user for an unban then that's an issue in it's own way - that would mean their peak rank on their profile would already appear as whatever they peaked while cheating 😳